arch/cpu/msp430/Makefile.msp430

```diff
@@ -44,6 +44,8 @@ CONTIKI_SOURCEFILES        += $(CONTIKI_TARGET_SOURCEFILES)
 
 # LTO is available in GCC 4.7.2, but crashes for MSP430.
 NO_LTO_TARGET = 1
+# No -fanalyzer before GCC 10.
+CFLAGS_FANALYZE =
```

Comment:
Is it possible to rather set `ANALYZE` to 0 as opposed to emptying the flags? As is, wouldn't a build for msp430 show `ANALYZE=1` but there would be no static-analysis?

Reply:
This is addressed, but it's now a platform action skip instead. For some reason, the output when running clang --analyze isn't binary code, it's the report.

Reply:
Reworked again, Clang Static Analyzer ignores -Werror so that needs to run through clang-tidy. This is now just an addition to CFLAGS for platforms that has a GCC that supports it.